### DISCUSSION TITLE
Clear my doubts

### DISCUSSION THREAD

**Discussion by Palak01gupta** [github/india#407] [General]
Hey , i am an intern here and is very new to all these platform. kindly help me to understand these platforms. i have started my first course based on azure , so shall i continue that readings and videos or do i have to do something on azure along with that? Also, what are we required to do on github?

**Comment by AbhishekMondal1** [comment]
You will get lab tasks time to time while reading the azure course. That time you have to login the Azure Sandbox environment and complete those tasks as directed.

**Reply by Palak01gupta** [reply]
okay and what about the github? how do we have to use this platform? and when do i have to claim the student pack . Also,tell me about the project synopsis, exactly what are we required to do in that?

**Reply by AbhishekMondal1** [reply]
- After completing the first module in **Describe Core Azure Concepts** you will be able to claim the pack. Check your cubicle's self learning section, there will be a side menu for claiming Student developer pack. open it and claim for the pack.
- I am also not sure how to use the github here as I have not encountered with any requirement for github while reading the courses. 
- submitting Project synopsis will start from 20th oct probably, so I think we have to develop our projects and push the code into github. also there is a github course at last section. So may be we have to work on github their also.
- I am not sure about project synopsis also. What they said earlier we have to write project synopsis on the basis of what we are learning about cloud. Don't know if they will give some topic for project or we have to find ourself any topics.

**Reply by Palak01gupta** [reply]
okay, thank you for this . i got a better vision on this.
Dont we have any mentors for guiding us?

**Reply by TheSuranaverse** [reply]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/ceMXzhfaka
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Reply by AbhishekMondal1** [reply]
I have seen some mentors here in the discuss forum. They were clearing some of the doubts, but don't know if they will guide us in projects or in courses.

**Reply by debopriyabasu** [reply]
do we have to pay for the AZ exams??

**Reply by Palak01gupta** [reply]
> do we have to pay for the AZ exams??

No ,there wont be any exam

**Reply by Palak01gupta** [reply]
@AbhishekMondal1 Are you able to understand this course?

**Reply by AbhishekMondal1** [reply]
Yes!

**Reply by AbhishekMondal1** [reply]
Which exercise are u doing?

**Reply by Palak01gupta** [reply]
test the database

**Reply by AbhishekMondal1** [reply]
you don't need any subscription for this exercise. In some of the upcoming exercise azure you need azure subscription. If we get the github pro then may be we can claim azure subscription with some credits. Otherwise we can do those exercise, we can just read and go ahead.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by Palak01gupta** [comment]
![Screenshot (100)](https://user-images.githubusercontent.com/91492424/137592703-99ca8d22-3654-458c-9e70-7cc02db98ba3.png)
i am not able to go ahead of this. what to do?

**Reply by AbhishekMondal1** [reply]
try to change the location to central india and check it

**Reply by AbhishekMondal1** [reply]
and i think the password is wrong

**Reply by Palak01gupta** [reply]
> try to change the location to central india and check it

from where?

**Reply by AbhishekMondal1** [reply]
> > try to change the location to central india and check it
> 
> from where?

In the Server creation Section. Delete the currnt server and create again with different location.

**Reply by Palak01gupta** [reply]
ok

**Reply by Palak01gupta** [reply]
now i have to do this from the very first step,right?

**Reply by Palak01gupta** [reply]
now its not even showing the resource group name by default

**Reply by AbhishekMondal1** [reply]
how much time is left in your sandbox?

**Reply by Palak01gupta** [reply]
its over now, i have started with 2nd out of 10

**Reply by AbhishekMondal1** [reply]
may be its got expired

**Reply by AbhishekMondal1** [reply]
Start it again

**Reply by Palak01gupta** [reply]
doing ..hehe

**Reply by AbhishekMondal1** [reply]
![2021-10-16 22_16_24-Window](https://user-images.githubusercontent.com/71382408/137595714-1a8f4e1b-5140-4dfb-bd3d-5ec36f2f9b00.jpg)
possibly u had typed wrong password while creating the db. I just deployed it and its working as per the instructions.

**Comment by Palak01gupta** [comment]
No, the password is what mentioned in steps.

**Reply by AbhishekMondal1** [reply]
Did u copied from or typed the passwd

**Reply by AbhishekMondal1** [reply]
In that above screenshot I can only see 11 characters but the actual paswd has 12 charactrs

**Reply by Palak01gupta** [reply]
did both

**Reply by Palak01gupta** [reply]
> In that above screenshot I can only see 11 characters but the actual paswd has 12 charactrs

i did the whole step again

**Comment by Palak01gupta** [comment]
Thanks for helping 😅

**Comment by Palak01gupta** [comment]
I checked many times the password , maybe the sandbox expired in the
middle.i didn't notice

**Comment by Palak01gupta** [comment]
Is the same going with you as well? Like i have completed 1st course but it shows in progress for 1st module and not started for rest. Also, i uploaded microsoft skills in linkedin and claimed for developer pack. what shall i do next? @AbhishekMondal1

**Reply by AbhishekMondal1** [reply]
- I have completed az900 long ago still it's showing in progress in that 1st section. In the Az104 only the first module is showing complete like any other interns. FRT website is not syncing our progress correctly.
- I have also did same for github dev pack but no response from them. They will not provide dev pack from FRT claim I think. You have to go manually and apply github dev pack from official github website. I am applying but they are rejecting giving different different causes.

**Reply by Palak01gupta** [reply]
okay

**Comment by Palak01gupta** [comment]
hey, i am not able to start with my second course . also , my first course is on level six which is incomplete although i can completed the entire course . What to do?

**Reply by AbhishekMondal1** [reply]
second course means **Azure Administrator (AZ-104)** ? What is the error showing show the screenshot. This course will redirect you to **futureskillprime** website from there you will be redirected to microsoft. 
Did you completed the last course **Describe Azure Cost Management And Service Level Agreements** of az900 module?

**Reply by Palak01gupta** [reply]
yeah, i mean AZ-104 only. yeah i have done the entire course 
![Screenshot (101)](https://user-images.githubusercontent.com/91492424/139028594-187b26d8-30a0-4a1a-8629-da5e3b53eef9.png)

**Reply by AbhishekMondal1** [reply]
did you logged in at futureskills? If not then first login and try to open this link.

**Reply by Palak01gupta** [reply]
the page is ot getting loaded , is future skills and future skill prime same?

**Reply by AbhishekMondal1** [reply]
Yes, i said futureskills for futureskillsprime. Have u tried to open this site in ur phone?

**Reply by konduyx18** [reply]
> yeah, i mean AZ-104 only. yeah i have done the entire course ![Screenshot (101)](https://user-images.githubusercontent.com/91492424/139028594-187b26d8-30a0-4a1a-8629-da5e3b53eef9.png)

Hi, how did you solve the problem? I am having the same issue? Thanks!

**Reply by AbhishekMondal1** [reply]
That's strange. When u clicked the course link, does it redirected to login page or it's showing this error page?

**Reply by Palak01gupta** [reply]
its redirected to the error page , the one i showed you in the ss. Also if i try to sign in from new tab to the site of future skills prime it also shows me the same page,the error one .

**Reply by AbhishekMondal1** [reply]
did u applied for github pack? try to get it as soon as possible.

**Reply by AbhishekMondal1** [reply]
@konduyx18 do u have the active azure student subscription? If u have then try to create a windows vm RDP then login there and check if the error is still showing or not.

**Reply by Palak01gupta** [reply]
> did u applied for github pack? try to get it as soon as possible.
I have clained it from the FRT portal itself 4-5 days ago but didnt get it yet. How to do it?

**Reply by AbhishekMondal1** [reply]
FRT is not giving the pack, no need to apply there. Go to the github student pack website directly and apply directly from there. Select ur email, submit ur collg id card and wait for few hours.

**Comment by kavin5221** [comment]
Hi guys, I have just started with the intern today. Can someone help me out on how to proceed further

**Reply by Palak01gupta** [reply]
wait for thee orientation, till then you can watch your industry sessions.

**Reply by kavin5221** [reply]
Any idea when is the next orientation session? Are you guys in Batch 2
I guess the previous orientation session was on 29th oct.

**Comment by konduyx18** [comment]
Thanks a lot for your help. I have an azure account but I don't know how to create a windows VM RDP and login there. could you please help me? Thanks again for your time!
________________________________

**Reply by AbhishekMondal1** [reply]
Look at this documentation [https://docs.microsoft.com/en-us/azure/virtual-machines/windows/quick-create-portal](url)

1. go to your azure portal
2. create resource
3. create virtual machine
4. select windows 2019 datacenter or any other windows image
5. fill the basic tab as per the documentation above
6. then review and create.
7. then go to connect and select RDP.
8. download the RDP file and connect with the username and password you used to create the VM.
9. then try to login your FRT account and open the az104 tutorial.
10. Check if it is working or not there.
11. after everything is done go to resource group and delete the VM along with the resource u have created, else your money will be charged from your azure student remaining balance.

**Comment by Palak01gupta** [comment]
my AZ-104 is not starting what to do?

**Reply by AbhishekMondal1** [reply]
That error looks like may be your isp blocked the website or may be other something browser related issue. So if its possible to login from some other place or other network it will be good to understand the reason of error. If u can login using some rdp then that could identify if its network related error or ur account related.

**Comment by Palak01gupta** [comment]
The problem is that our college has not issued us the college id card yet.
So, I don't have that

**Reply by AbhishekMondal1** [reply]
Do u have any online result or syllabus which is online in your college/university student portal. Then u can submit that screenshot for verification. Check what documents are valid in the github pack website and submit that which is possible.

**Reply by Palak01gupta** [reply]
okay

**Comment by konduyx18** [comment]
Hi Abhishek,
Thanks again for your help. I followed your instructions, but I have an error when clicking the RDP file. If possible could you please tell what should I do to solve the problem? Take care.
 [cid:931922f8-ef2e-4c5b-b1ba-41b4b84c41b3]
there is no place to enter a password and username also  I can only click the connect button
[cid:ea627b16-67e5-4c07-a7f0-45d8a28fb17f]

________________________________

**Reply by AbhishekMondal1** [reply]
Is it trying to connect to the ip address of your VM's ip address? First it will check if its running or not, then it will ask for permission and will show the login screen. If its not connecting after creating the VM try after 2-3 min.

**Comment by anikettyagi13** [comment]
Hi guys! My progress is not updating. Can anyone help me with it?
